frontend/ui/src/components/organization/tree-item-row.tsx

```
@@ -0,0 +1,325 @@
+          <Checkbox />
+        </TableCell>
+        <TableCell className="border border-gray-200">
+          <div
+            className="flex items-center"
+            style={{ paddingLeft: `${level * 24}px` }}
+          >
+            {item.type === "folder" && (
+              <button
+                onClick={() => onToggleExpand(item.id)}
+                className="mr-2 focus:outline-none"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)